spec/lib/disability_compensation/providers/document_upload/evss_supplemental_document_upload_provider_spec.rb

```diff
@@ -72,4 +72,60 @@
       end
     end
   end
+
+  describe 'logging methods' do
+    # We don't want to generate an actual submission for these tests,
+    # since submissions have callbacks that log to StatsD and we need to test
+    # only the metrics in this class
```

Comment:
That's interesting. Do we have other tests we might need to change? I took a quick look and didn't see any. Just checking to see if you had any in mind.

Reply:
Not that I'm aware of, this would apply in any case where we are checking a StatsD increment but also creating a submission; the tests for StatsD look like they are all checking for a specific metrics key, so they would be failing if Rspec is detecting the other StatsD metric increment in the test run
<img width="295" alt="Screenshot 2024-08-19 at 11 21 44 AM" src="https://github.com/user-attachments/assets/8cb2e5c1-f589-496b-a719-00b6888700e8">